docs/admin/integrations/ipfabric_setup.md

```diff
@@ -57,6 +57,7 @@ PLUGINS_CONFIG = {
 | `ipfabric_safe_delete_location_status` | The status that is set for a Location when the `Safe Delete Mode` flag is set in the Job. | `Decommissioning` |
 | `ipfabric_safe_delete_vlan_status` | The status that is set for a VLAN when the `Safe Delete Mode` flag is set in the Job. | `Deprecated` |
 | `ipfabric_safe_delete_ipaddress_status` | The status that is set for an IP Address when the `Safe Delete Mode` flag is set in the Job. | `Deprecated` |
+| `ipfabric_elongate_interface_name` | Whether to attempt to elongate interface names as found in IP Fabric. | `False` |
```

Comment:
why not keep use `canonical_interface_name`? I don't love either name, but at least it's consistent.